### PR TITLE
Add make commands

### PR DESCRIPTION
**Pull request by kchygoe**
- install
- install-gpu
## What this patch does to fix the issue.
To develop on local env

## Link to any relevant issues or pull requests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,6 +10,20 @@ deps:
...
+	pip install pycocotools==2.0.0
+
+.PHONY: install-gpu
+install-gpu: install
+	pip install -e .[gpu]
+	pip install -e .[dist]
+
+.PHONY: lint
+lint:
+	flake8 ./blueoil --exclude=templates,converter
```
Comment (tfujiwar) [inline Makefile]: Isn't `test-blueoil-pep8` enough?

Comment (kchygoe) [reply Makefile]: 1st, this is for development.
2nd, for flake8, using docker sounds too complecated

Comment (tfujiwar) [reply Makefile]: I see.

**Pull request merged**